src/components/Form/index.tsx

```diff
@@ -30,16 +32,20 @@ export const StyledInput = styled.input`
 
 const StyledTextarea = styled.textarea`
   font-size: 1.3rem;
-  padding: 10px 10px 10px 5px;
-  background: #cfd8dc;
+  padding: 2rem 1rem 0.5rem 1rem;
+  background: white;
```

Comment:
let's keep #fff for consistency https://github.com/aretecode/konsilos/pull/11/commits/2a40c3b6ce7916777ea757e238861f4ca1894470